ecosystem/sep-0012.md

```diff
@@ -52,7 +64,6 @@ The fields below should be placed in the request body using the `multipart/form-
 Name | Type | Description
 -----|------|------------
 `account` | `G...` string | The Stellar account ID to upload KYC data for
-`jwt` | string | The JWT previously sent by the anchor via the [SEP-10](sep-0010.md) authentication flow
```

Comment:
Do we need to delete this line, now that we're keeping the `jwt` param support?

Reply:
Well, I thought about putting it back in but I thought since we are recommending against it, even putting it in with an (optional) might make people think that it is simply optional where really it is not recommended.